recipes/Getting_Started/Getting_Started_with_Ollama.ipynb

```diff
@@ -18,18 +18,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Install Ollama\n",
+    "The Ollama server can be run locally on your system or in the notebook itself.\n",
     "\n",
-    "_Option A: Package Installer (MacOS, Windows)_\n",
+    "### Option A: Running Locally (Linux, MacOS, Windows)\n",
     "\n",
-    "[Download from Ollama.com](https://ollama.com/download) and install "
+    "1. [Download and install Ollama](https://ollama.com/download), if you haven't already.\n",
+    "1. Start the Ollama server: `ollama serve`\n",
+    "1. Pull down Granite models:\n",
+    "\n",
+    "    ```shell\n",
+    "    ollama pull granite-code:3b\n",
+    "    ollama pull granite-code:8b\n",
+    "    ollama pull granite-code:20b\n",
+    "    ```"
```

Comment:
I like the refactoring you've done here: putting the full instructions under each option.